src/components/Dropdown/Presentation.tsx

```diff
@@ -4,33 +4,42 @@ import { Dispatch, SetStateAction } from "react";
 
 import styles from "./dropdown.module.css";
 import { dropdownLinksType } from "./types/dropdownLinks";
+import React from "react";
 
-
-export function DropdownPresentation({ logout, dropdownLinks, setIsDropdownVisible } : {
-    logout: () => void;
-    dropdownLinks: dropdownLinksType[];
-    setIsDropdownVisible: Dispatch<SetStateAction<boolean>>;
+export function DropdownPresentation({
+  dropdownLinks,
+  setIsDropdownVisible,
+}: {
+  dropdownLinks: dropdownLinksType[];
+  setIsDropdownVisible: Dispatch<SetStateAction<boolean>>;
 }) {
-
-    const dropdownItems = dropdownLinks.map((link) => {
-        return (
-        <ListItem key={link.id} className={styles.dropdown_items}>
-            <Link href={link.link} className={styles.dropdown_links}>
-                {link.name}
+  const dropdownItems = dropdownLinks.map((item) => {
+    return (
+      <React.Fragment key={item.id}>
+        {typeof item.label !== "string" ? (
+          <>
+            <hr className={styles.line} />
+            {item.label()}
```

Comment:
- why are we calling label here can this be `item.label` instead ?

Reply:
it was not called in dropdown link so that's why i called here will change it